Makefile

```diff
@@ -2,7 +2,7 @@
 EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
 OUTPUT_DIRNAME		?= output/
 DOC_FILENAME		?= oci-runtime-spec
-SHELL			?= $(shell command -v bash 2>/dev/null)
+SHELL			?= $(shell command -v sh 2>/dev/null)
```

Comment:
fixed it

On Mon, Aug 29, 2016 at 2:06 PM W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/runtime-spec/pull/547#discussion_r76655651>
> :
>
> > @@ -2,7 +2,7 @@
> >  EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
> >  OUTPUT_DIRNAME		?= output/
> >  DOC_FILENAME		?= oci-runtime-spec
> > -SHELL			?= $(shell command -v bash 2>/dev/null)
> > +SHELL			?= $(shell command -v sh 2>/dev/null)
>
> I don't think we need this. From the GNU Make docs
> <https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html>
> :
>
> The program used as the shell is taken from the variable SHELL. If this
> variable is not set in your makefile, the program /bin/sh is used as the
> shell.
>
> Although GNU Make does some dancing for DOS/Windows. Are you including
> this line to skip that dance? If so, why?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/547/files/300eb002e587ff9851578f9147b438cc53c9b5d0#r76655651>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6Wsldb7cZg5PLBbm9ycIlQ_OtVrrks5qkx-SgaJpZM4JvfF1>
> .
>